api/datastore/sql/sql.go

```diff
@@ -51,12 +51,13 @@ var tables = [...]string{`CREATE TABLE IF NOT EXISTS routes (
 	headers text NOT NULL,
 	config text NOT NULL,
 	annotations text NOT NULL,
-	created_at text,
+	created_at varchar(256),
```

Comment:
i think this will break existing db for mysql at least :( - can you omit? have a note to add a migration to do this, it's not very straightforward from what I can tell.